addons/pos_restaurant/static/src/css/restaurant.css

```diff
@@ -315,3 +315,80 @@
     margin-right: 4px;
 }
 
+/* ------ SPLIT BILL SCREEN ------- */
+
+/* ------ WIDER SCREEN ------ */
+@media screen and (min-width: 768px) {
```

Comment:
you should put the media tags after the definitions of standard rules. Because if you have similar attributes set in the media specific rules, it won't work.

Example I am in mobile:
if I have:
@media screen and (max-width: 768px) {
.splitbill-screen.screen .main {
       height: 50%;
    }
}
.splitbill-screen.screen .main {
       height: 100%;
}
=> the height will always be 100%

If I reverse the order:


.splitbill-screen.screen .main {
       height: 100%;
}
@media screen and (max-width: 768px) {
.splitbill-screen.screen .main {
       height: 50%;
    }
}

=> the height in mobile will be 50%

Reply:
got it.